modules/administration/pages/ssl-certs-selfsigned.adoc

```diff
@@ -14,66 +20,12 @@ Before you begin, ensure you have:
 Your key and certificate files must be in PEM format.
```

Comment:
This section is not correct in this chapter. We re-generate all of it, so you do not need them beforehand.
This is only important in the "Import" case

Reply:
Line 14 - 20 can be removed